### ISSUE TITLE
An error when creating new instance

### ISSUE THREAD

**Issue by pavolve** [ajimix/Input-Framer#4]
`input = new InputModule.Input`

always return an error "TypeError: undefined is not an object (evaluating 'this.input.style')"
Inspector shows input.coffee line 12

Framer Studio v70

**Comment by ajimix** [comment]
Hello @pavolve I just tried with v70 and works fine for me, here you have a working demo: http://share.framerjs.com/3pv079i24nmj/

Could you please share a link like I did with a project that shows the error? Maybe that way I can help you out :)

**Comment by pavolve** [comment]
Sure http://share.framerjs.com/5q7hs5z4l9a2/

**Comment by pavolve** [comment]
Sure http://share.framerjs.com/5q7hs5z4l9a2/

**Comment by ajimix** [comment]
That's weird, there seems to be an error with your Project. I created a new project, and manually copied the resources app.coffee, modules and imported folders and now it works fine as can be seen here: http://share.framerjs.com/4zxdmk665vh2/

I think there was some kind of Bug with framer... I would suggest that you open an issue with them and you submit both links, your project not working and the one I just submitted which works and both have exactly the same code. They will be able to fix the bug hopefully.

I'm closing this issue as it's not related with the module but with Framer itself.
